docs/developers/references/traits/autoscale.md

```diff
@@ -0,0 +1,100 @@
+# Automatically scale workload by cron or resource utilization
+
+## Prerequisites
+ - [ ] [KEDA v2.0 Beta](https://keda.sh/blog/keda-2.0-beta/)
+ 
+   KEDA will be automatically deployed during veal installation, so just run the following command.
+   ```shell
+   $ vela install
+   ```
+   
+   You can also double check whether KEDA is successfully installed.
+   ```
+   $ helm list -n keda
+   NAME	NAMESPACE	REVISION	UPDATED                             	STATUS  	CHART             	APP VERSION
+   keda	keda     	1       	2020-11-02 17:01:16.122247 +0800 CST	deployed	keda-2.0.0-beta1.2	2.0.0-beta
+   ```
```

Comment:
please don't expose so much details for developers. I think we should remove this section.

Or we should using `vela traits` to provide status whether the helm background is ready or not.

```
$ vela traits
NAME           STATUS  DESCRIPTION
autoscale    Ready     Automatically scale workloads by cron or resource utilization
```

Reply:
+1, thanks.